etc/notes/CHANGES_6.0.0.md

```diff
@@ -2,23 +2,339 @@
+
+### `db.addUser()` and `admin.addUser()` removed
+
+The deprecated `addUser` APIs have been removed. The driver maintains support across many server versions and the `createUser` command has support for different features based on the server's version. Since applications can generally write code to work against a uniform and perhaps more modern server, the path forward is for applications to send the `createUser` command directly.
+
+The associated options interface with this API has also been removed: `AddUserOptions`.
+
+See `createUser` documentation: https://www.mongodb.com/docs/manual/reference/command/createUser/
+
+```ts
+const db = client.db('admin');
+// Example addUser usage
+await db.addUser('myUsername', 'myPassword', { roles: [{ role: 'readWrite', db: 'mflix' }] });
+// Example equivalent command usage
+await db.command({
+  createUser: 'myUsername',
+  pwd: 'myPassword',
+  roles: [{ role: 'readWrite', db: 'mflix' }]
+});
+```
+
+### `collection.stats()` removed
+
+The `collStats` command is deprecated starting in server v6.2 so the driver is removing its bespoke helper in this major release. The `collStats` command is still available to run manually via `await db.command()`. However, the recommended migration is to use the `$collStats` aggregation stage. You can find more information here: [$collStats](https://www.mongodb.com/docs/upcoming/reference/operator/aggregation/collStats/#mongodb-pipeline-pipe.-collStats).
+
+The following interfaces associated with this API have also been removed: `CollStatsOptions` and `WiredTigerData`.
+
+### `BulkWriteResult` deprecated properties removed
+
+The following deprecated properties have been removed as they duplicated the functionality provided by the common driver spec compliant properties. The list indicates what properties provide the correct migration:
+
+- `BulkWriteResult.nInserted` -> `BulkWriteResult.insertedCount`
+- `BulkWriteResult.nUpserted` -> `BulkWriteResult.upsertedCount`
+- `BulkWriteResult.nMatched` -> `BulkWriteResult.matchedCount`
+- `BulkWriteResult.nModified` -> `BulkWriteResult.modifiedCount`
+- `BulkWriteResult.nRemoved` -> `BulkWriteResult.deletedCount`
+- `BulkWriteResult.getUpsertedIds` -> `BulkWriteResult.upsertedIds` / `BulkWriteResult.getUpsertedIdAt(index: number)`
+- `BulkWriteResult.getInsertedIds` -> `BulkWriteResult.insertedIds`
+
+### Deprecated SSL options have been removed
+
+The following options have been removed with their supported counterparts listed after the ->
+
+- `sslCA` -> `tlsCAFile`
+- `sslCRL` -> `tlsCRLFile`
+- `sslCert` -> `tlsCertificateKeyFile`
+- `sslKey` -> `tlsCertificateKeyFile`
+- `sslPass` -> `tlsCertificateKeyFilePassword`
+- `sslValidate` -> `tlsAllowInvalidCertificates`
+- `tlsCertificateFile` -> `tlsCertificateKeyFile`
+
+### The deprecated `keepAlive` and `keepAliveInitialDelay` options have been removed
+
+TCP keep alive will always be on and now set to a value of 30000ms.
+
+## 🪦 Removal of "dead" code
```

Comment:
Broken unicode character here

Reply:
Windows 10 doesn't support this emoji. Maybe: 🗑️?

Reply:
But the gravestone is so perfect... Just kidding, open to alternate suggestions of course

Reply:
Check now?

Reply:
There's also ⚰️ if we're taking "dead" code literally

Reply:
Confirmed that the trash can emoji shows up on windows 👍🏿